src/modules/universal-discovery/components/selected-content/css/selected.content.popup.component.css

```diff
@@ -1,12 +1,28 @@
 .c-selected-content-popup .c-popup {
-    width: 640px;
+    width: fit-content;
```

Comment:
According to caniuse.com Edge doesn't support fit-content

Reply:
we support Edge